src/objects/core/zcl_abapgit_file_status.clas.abap

```diff
@@ -510,6 +512,29 @@ CLASS ZCL_ABAPGIT_FILE_STATUS IMPLEMENTATION.
       MOVE-CORRESPONDING <ls_res1> TO ls_file.
     ENDLOOP.
 
+    " Check if namespaces exist already
```

Comment:
suggest extracting the logic to a new private method

Reply:
refactored